src/deterministic_secure_broadcast.rs

```diff
@@ -162,6 +183,9 @@ impl<A: SecureBroadcastAlgorithm> SecureBroadcastProc<A> {
         if self.validate_packet(&packet) {
             self.process_packet(packet)
         } else {
+            println!("[DSB/INVALID] packet failed validation: {:?}", packet);
```

Comment:
in sync_from():

> self.peers.extend(state.peers);

This is why I was thinking we need to use Orswot.  Because that extend() will only aggregate, never remove.   If that is not a problem, then I think the extend() call at least needs an explanatory comment about it.

Reply:
When we first implement anti-entropy, we'll be replicating packets (not state), we'll replay these packets onto the local state which should take care of removing/adding peers in the correct order.